src/api/prepareCommit.js

```
@@ -0,0 +1,106 @@
+      })
+    } catch {
+      // Empty repo, no HEAD
+    }
+
+    const files = await _walk({
+      fs,
+      cache,
+      dir,
+      gitdir: updatedGitdir,
+      trees: [STAGE()],
+      map: async function(filepath, [stage]) {
```

ESLint

[error] Insert `·`

(prettier/prettier)